client/src/components/Upload/DefaultBox.vue

```diff
@@ -99,7 +100,16 @@ const listExtensions = computed(() => props.effectiveExtensions.filter((ext) =>
 const showHelper = computed(() => Object.keys(uploadItems.value).length === 0);
 const uploadValues = computed(() => Object.values(uploadItems.value));
 
-const queue = computed(() => createUploadQueue());
+onMounted(() => {
+    queue.value = initUploadQueue();
+});
```

Comment:
To avoid duplication, you can put `{ immediate: true }` in the watcher options

Reply:
I keep forgetting that, thanks!

Reply:
There's also this neat [shorthand](https://vueuse.org/shared/watchImmediate/)